web/viewer.css

```diff
@@ -1559,13 +1453,261 @@ dialog :link {
   display: none !important;
 }
 
-@media all and (max-width: 900px) {
-  #toolbarViewerMiddle {
-    display: table;
-    margin: auto;
-    left: auto;
-    position: inherit;
-    transform: none;
+.toolbarLabel {
+  width: max-content;
+  min-width: 16px;
+  height: 100%;
+  padding-inline: 4px;
+  margin-inline: 4px;
```

Comment:
This was changed in the latest update, related to https://github.com/mozilla/pdf.js/pull/18385#issuecomment-2369296425, however that also affects the positioning e.g. the `numPages` label and potentially *other* elements as well.
Maybe that shouldn't be changed unconditionally like this, but only for elements in the findbar?